ui/insight/src/info.js

```diff
@@ -15,7 +15,7 @@ module.exports = function (ctrl) {
       ),
     ]),
     m('div.content', [
-      m('p', ['Insights over ', m('strong', ctrl.user.nbGames), ' rated games.']),
+      m('p', trans.insightsOverNbRatedGames, m('strong', ctrl.user.nbGames)),
```

Comment:
I think this should probably be `m('p', trans.insightsOverNbRatedGames.vdom(m('strong', ctrl.user.nbGames)))`?

Reply:
Thanks!